sass/lib/_colours.scss

```diff
@@ -6,4 +6,8 @@ $nw-c-japanese-laurel: #00a400;
 $nw-c-grenadier: #d71e00;
 $nw-c-forest-green: #198627;
 $nw-c-nobel: #b3b3b3;
-$nw-c-tapa: #78746e;
\ No newline at end of file
+$nw-c-tapa: #78746e;
+$nw-c-white: #fff;
+$nw-c-mine-shaft: #222;
+$nw-c-tundora: #404040;
```

Comment:
this will overwrite https://github.com/bbc/gs-sass-tools/blob/master/settings/_news-colours.scss#L22